qdrant-landing/content/articles/dedicated-service.md

```diff
@@ -0,0 +1,152 @@
+As a result, running a dedicated vector search engine might be even cheaper, as it allies optimization specifically for vector search use cases.
+
+For instance, Qdrant implements a number of [quantization techniques](documentation/guides/quantization/) that can significantly reduce the memory footprint of embeddings.
+
+Not to mention that data transfer costs within the same region are usually free in the majority of cloud providers.
+
+
+###### What is more seamless than your current database adding vector search capability?
+
+
+In contrast to the short-term attractiveness of integrated solutions, dedicated search engines propose flexibility and a modular approach.
+You don't need to update the whole production database each time some of the vector plugins are updated.
+Maintenance of a dedicated search engine is as much isolated from the main database as the data itself.
+
+In fact, integration of a bit more complex scenatios, such as read/write segregation, is much easier with a dedicated specialized solution.
+You can easily build cross-region replication, to ensure low latency for your users.
+
+{{< figure src=/articles_data/dedicated-service/region-based-deploy.png caption="Read/Write segregation + cross-regional deployment" width=80% >}}
+
+It is especially important in large enterprise organizations, where the responsibility for different parts of the system is distributed among different teams.
+In those situations, it is much easier to maintain a dedicated search engine for the AI team than to convince the core team to update the whole primary database.
+
+Additionally, you won't depend on sluggish development cycles of huge integrated products, which have to consider legacy support and backward compatibility for all their features.
+
+
+###### Databases can support RAG use-case end-to-end.
+
+Putting aside performance and scalability questions, the whole discussion about implementing RAG in the DBs assumes that the only detail missing in traditional databases is the vector index and the ability to make fast ANN queries.
+
+In fact, we believe that the capabilities of vector search have only started to unwrap.
+For example, in our recent article, we discuss the possibility of building an [exploration API](/articles/vector-similarity-beyond-search/) to fuel the discovery process - an alternative to kNN search, where you don’t know what exactly you are looking for.
+
+
+## Summary
+
+Large-scale production systems usually consist of different specialized services for a good reason.
+We discussed those of them that are related to vector search use cases in this article, but there are many more.
+
+Scaling even the simplest systems unlocks a whole new level of challenges with each order of magnitude, and that is especially true for vector search.
+Having control and flexibility over each part of the system is crucial for controlling those challenges and, hence, the cost of the system.
```

Comment:
There needs to be some sort of stronger final paragraph bringing this all together.